StripePaymentSheet/StripePaymentSheet/Source/Internal/API Bindings/Link/STPAPIClient+Link.swift

```diff
@@ -330,7 +335,7 @@ extension STPAPIClient {
         let parameters: [String: Any] = [
             "credentials": ["consumer_session_client_secret": consumerSessionClientSecret],
             "request_surface": "ios_payment_element",
-            "types": ["card"],
+            "types": supportedPaymentDetailsTypes.map(\.rawValue),
```

Comment:
Why do we use a Set instead of an Array for supportedPaymentDetailsType? It's a little clunkier in Swift, and I'm not sure if the underlying JSON response can really represent a Set. It was causing issues with ordering in our tests: https://github.com/stripe/stripe-ios/pull/4549

Reply:
I’ll change it to a list, now that I’ve learned about the test issues it can cause.